client/Makefile

```diff
@@ -30,12 +30,12 @@ grpc: $(PROTOBUF_OUTPUT)
 syntax: ${PROTOBUF_OUTPUT}
 	flake8 `git ls-files '**.py'`
 	mypy -p api
-	mypy -p core
-	mypy -p unit_tests
+	mypy -p zeth.core
+	mypy -p zeth.test
+	mypy -p zeth.cli
 	mypy -p test_commands
-	mypy -p zeth_cli
-	mypy -p zeth_misc_cli
-	pylint core zeth_cli unit_tests test_commands zeth_misc_cli
+	mypy -p zeth_misc
```

Comment:
> Ideally everything, including this and test_commands, would be in the zeth package.

If I understood your point correctly, you initially wanted to refactor the client code structure to make it easier to use the `zeth` python package in other projects. Why would we want to include the `test_commands` and `zeth_misc` as part of this package? To me these things use the `zeth` package but stand on their own.

> I'm wondering if there is a better name. 

Yes probably. I just wanted to move away from `debug` because this CLI is not used for debug purposes. `zeth_helper` sounds good to me.

Reply:
> zeth_helper sounds good to me.

That seems like a reasonable compromise.

> Why would we want to include the test_commands and zeth_misc as part of this package? To me these things use the zeth package but stand on their own.

Sorry ... `test_commands` should indeed be outside of the `zeth` module, with `test`.  I later reaslised that `test` needs to be *outside* the main module.

So, to be clear, I'm suggesting:
```
client/
  zeth/
    api/
    core/
    cli/
    helper/
  test/
  test_commands/
```

For `helper` it's already part of the package (can't be installed seperately, and indeed may be useful for users of this package), so it seems cleaner to have everything in the `zeth` module. But I guess it could be kept as a "subpackage".  It just means the `zeth` package contains `zeth` and `zeth_helper` modules, rather than just `zeth`.

Reply:
As discussed offline, helper had a dependency on test_commands which has been removed now. Resolving.